### PR TITLE
Api end points

### PR DESCRIPTION
**Pull request by bvenceslas**

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -84,16 +87,15 @@ router.get('/transactions/:tid', (req, res) => {
 // post a transaction
 router.post('/transactions', (req, res) => {
 
-    const {montant, type, date, uid} = req.body;
+    const {montant, somme, type, date, uid_sender, uid_receiver} = req.body;
```
Comment (guillainbisimwa) [inline routes/api.js:90]: 'Montant` and `somme`?

File: routes/api.js [hunk 2]
```diff
@@ -84,16 +87,15 @@ router.get('/transactions/:tid', (req, res) => {
...
 
     newTransaction.save()
         .then(transactionSaved => res.json(transactionSaved))
         .catch(err => res.status(400).json({error_message:err}));
 });
 
-// register a unser
-
+// register a user
 router.post('/register', (req, res) => {
```
Comment (guillainbisimwa) [inline routes/api.js:99]: Imaginer dans `register` on pas prevus bcp de details comme 

- [x]  url profile
- [x]  Adresse de livraison par defaut
- [x] sex
On va ajouter ces elements dans l'app mais ne seront pas obligatoires

**Review by guillainbisimwa — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello Josh :wave:

__You've made excellent progress with nice implementation models and API :+1: 
However, some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_I hope all the `dates` will be automatically added _

* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Guillain BISIMWA
[Twitter](https://twitter.com/gullain_bisimwa) | [Github](https://github.com/guillainbisimwa) | @tGuillain Bisimwa**
__Happy Coding :smiley:__

File: routes/api.js (hunk 2 above)
Comment (bvenceslas) [reply routes/api.js:99]: done, only verifier url_profile kama njo vile inapashwa kuwa

File: routes/api.js (hunk 1 above)
Comment (bvenceslas) [reply routes/api.js:90]: j'ai enlevé la somme

**Comment by bvenceslas** [comment]
img_documents

**Review by guillainbisimwa — APPROVED** [review]
# Status:  APPROVED  🎉 🚀
#### :clap: :clap:
#### Don't forget to merge and deploy to Heroku! 

gbisimwa

**Pull request merged**